contract/sep-nft.md

```diff
@@ -0,0 +1,61 @@
+## Preamble
+
+```
+CIP: 
+Title: NFT
+Author: Stellar Development Foundation <@stellar>
+Status: Draft
+Discussion: TBD
+Created: 2023-10-11
+Version 0.1.0
+```
+
+## Summary
+
+This proposal defines a standard interface that NFTs on Soroban powered
+networks, such as the Stellar network, can implement to interoperate with
+contracts that use NFTs, and Stellar assets.
```

Comment:
What is the plan for the Stellar assets interop?

Reply:
I made a PR to try to enable SAC interop.
https://github.com/vinamogit/stellar-protocol/pull/2